services/provider/server/server.go

```diff
@@ -420,6 +449,9 @@ func (s *OCSProviderServer) GetDesiredClientState(ctx context.Context, req *pb.G
 			inMaintenanceMode,
 			isConsumerMirrorEnabled,
 			topologyKey,
+			isRbdDriverEnabled,
```

Comment:
have you tested for disabling of any driver at runtime?

Reply:
We were only enabling drivers